### PR TITLE
feat(1668): add more information to email notifications

### PR DESCRIPTION
**Pull request by yoshwata**
## Context

Email notifications are too simple that we can't find what happened with at a glance.

## Objective

This PR add more information to email notifications.

Added informations
- changed files
- commit message
- commit sha with github link

This is sample that I confirm in my environment.
5IHpj5dccc8e773f93ee10b6fd36d

## References

https://github.com/screwdriver-cd/screwdriver/issues/1668

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by kumada626** [comment]
I think that it should better to leave a message when there are no changed files.

**Comment by yoshwata** [comment]
I fixed it. How is this?
BrPAb5dcd1449b8a995ea0bd0a18b

**Comment by kumada626** [comment]
Looks good. You have to write a test the case of no changed file.

**Comment by yoshwata** [comment]
Test added.

File: index.js [hunk 1]
```diff
@@ -111,15 +111,40 @@ class EmailNotifier extends NotificationBase {
...
+        let changedFilesStr = '';
+
+        if (changedFiles.length > 0 && changedFiles[0] !== '') {
+            changedFiles.forEach((file) => {
+                const li = '<li>{{contents}}</li>';
+
+                changedFilesStr += tinytim.tim(li, { contents: file });
+            });
+        } else {
+            changedFilesStr = 'There is no changed files.';
```
Comment (tkyi) [inline index.js]:
```suggestion
            changedFilesStr = 'There are no changed files.';
```

Comment (yoshwata) [reply index.js]: Thanks. I fixed.

**Pull request merged**